adb-sync

```diff
@@ -305,6 +306,20 @@ def BuildFileList(fs, path, prefix=b''):
     print('Note: unsupported file: %s' % path.decode('utf-8', errors='replace'))
 
 
+def FilterFileList(file_iter, excludes):
+    if not excludes:
+        excludes = []
+    for f in file_iter:
+        excluded = False
+        for pat in excludes:
+            if fnmatch.fnmatch(f[0], pat):
```

Comment:
Not sure if f[0] is right?

Reply:
thats correct, since its a tuple:
```
('/file/abc', posix.stat_result(st_mode=16877, st_ino=1, st_dev=0, st_nlink=1, st_uid=-2, st_gid=-2, st_size=None, st_atime=1528642980.0, st_mtime=1528642980.0, st_ctime=1528642980.0))
```

`f[0]` is the '/file/abc'